snippets/core/system-text-json/csharp/SerializeCustomEncoding.cs

```diff
@@ -0,0 +1,65 @@
+﻿// <SnippetUsings>
+using System;
+using System.Text.Encodings.Web;
+using System.Text.Json;
+using System.Text.Unicode;
+// </SnippetUsings>
+
+namespace SystemTextJsonSamples
+{
+    class SerializeCustomEncoding
+    {
+        public static void Run()
+        {
+            string jsonString;
+            var weatherForecast = WeatherForecastFactories.CreateWeatherForecastCyrillic();
+            weatherForecast.DisplayPropertyValues();
+
+            Console.WriteLine("Default serialization - non-ASCII escaped");
+            var options = new JsonSerializerOptions
+            {
+                WriteIndented = true
+            };
+            jsonString = JsonSerializer.Serialize(weatherForecast, options);
+            Console.WriteLine(jsonString);
+            Console.WriteLine();
+
+            Console.WriteLine("Serialize language sets unescaped");
+            // <SnippetLanguageSets>
+            options = new JsonSerializerOptions
+            {
+                Encoder = JavaScriptEncoder.Create(UnicodeRanges.Cyrillic, UnicodeRanges.GreekExtended),
```

Comment:
@tdykstra On .NET Core 3.1/5.0, this doesn't output the expected string (due to the fix in https://github.com/dotnet/corefx/pull/40787).

**I recommend changing the TFM of this project to target 3.1 (since that is the LTS release).**

The sample outputs the following on 3.1/5.0:
```text
{
  "\u0044\u0061\u0074\u0065": "2019-08-01T00:00:00-07:00",
  "\u0054\u0065\u006D\u0070\u0065\u0072\u0061\u0074\u0075\u0072\u0065\u0043\u0065\u006C\u0073\u0069\u0075\u0073": 25,
  "\u0053\u0075\u006D\u006D\u0061\u0072\u0079": "жарко"
}
```

The fix is to actually specify the BasicLatin unicode range as well.

Also is `UnicodeRanges.GreekExtended` needed here? Seems unnecessary.

```suggestion
                Encoder = JavaScriptEncoder.Create(UnicodeRanges.BasicLatin, UnicodeRanges.Cyrillic),
```

cc @steveharter, @layomia